src/Native/Runtime/amd64/InteropThunksHelpers.S

```diff
@@ -4,19 +4,43 @@
 
 .intel_syntax noprefix
 #include <unixasmmacros.inc>
-
-//TODO: Implement these stub method for non-windows
+#include <AsmOffsets.inc>         // generated by the build from AsmOffsets.cpp
+#define POINTER_SIZE 8
 
 LEAF_ENTRY RhpCommonStub, _TEXT
-    int 3
+
+    PUSH_ARGUMENT_REGISTERS
+
+    // +8 for alignment
+    alloc_stack    (SIZEOF_MAX_FP_ARG_SPILL + 8)
```

Comment:
Also, this constant seems to be meant for profiler support, so it seems it would be better to define a different constant locally in this file or in the unixasmmacrosamd64.inc (e.g. NUM_FP_ARG_REGISTERS and use NUM_FP_ARG_REGISTERS * 16 or something like that).